ext/re2/extconf.rb

```diff
@@ -277,7 +298,7 @@ def process_recipe(name, version)
       '-DCMAKE_INSTALL_LIBDIR=lib'
     ]
     recipe.configure_options += cmake_compile_flags(recipe.host)
-    delete_cmake_generator_option!(recipe.configure_options)
+    delete_cmake_generator_option!(recipe.configure_options) if MiniPortile.mingw? && !cmake_with_msys_generator?
```

Comment:
We can drop this once https://github.com/flavorjones/mini_portile/pull/129 is merged and released.